server/container/guice/cassandra-guice/src/test/java/org/apache/james/JamesCapabilitiesServerTest.java

```diff
@@ -70,21 +70,26 @@ public void startShouldSucceedWhenMoveCapability() throws Exception {
         server = createCassandraJamesServer(mailboxManager);
 
         server.start();
+    }
 
+    @After
+    public void teardown() {
         server.stop();
```

Comment:
it should not happen, so the test fails, but anyway, there's nothing to clean up, really